main.py

```diff
@@ -0,0 +1,23 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def join(*args, sep='-'):

```

Comment:
Please add documentation to this function.

Reply:
I did it.